gamedata.py

```diff
@@ -925,7 +925,7 @@ def score(self) -> int:
 
     @property
     @abstractmethod
-    def score_breakdown(self) -> list[str]:
+    def score_breakdown(self) -> str:
```

Comment:
I believe this should remain `list[str]`, as that's what the subclasses do

Reply:
oops, forgot to push this